File: table.go
package main

import "fmt"

/* function to be run by program */

func main() {

    /* default table info */

    var table int
    var table_start int
    var table_end int

    /* take user input for table */

    fmt.Print("Enter table: ")
    fmt.Scanf("%d", &table)

    fmt.Print("Enter table start: ")
    fmt.Scanf("%d", &table_start)

    fmt.Print("Enter table end: ")
    fmt.Scanf("%d", &table_end)
    
    /* generate table based on info */

    for table_term:=table_start; table_end>=table_term; table_term++ {
        fmt.Printf("%d x %d = %d\n", table, table_term, table * table_term)
    }
}

File: table.c
#include <stdio.h>
#include <stdbool.h>

/* some error defines */

#define ERROR       1
#define NO_ERROR    0

/* function to be run by program */

int main()
{
    /* default table info */

    int table;
    int table_start;
    int table_end;

    /* take user input for table info */

    printf("Enter table: ");
    scanf("%d", &table);

    printf("Enter table start: ");
    scanf("%d", &table_start);

    printf("Enter table end: ");
    scanf("%d", &table_end);

    /* generate table based on info */

    for (int table_term=table_start; table_end>=table_term; table_term++) {
        printf("%d x %d = %d\n", table, table_term, table * table_term);
    }

    /*  show no error at the end */

    return NO_ERROR;
}
